openapi.yaml

```diff
@@ -132,147 +467,202 @@ securityDefinitions:
     scopes:
       full: Full access to the API
 definitions:
-  Meta:
+  PizzasResults:
     properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
-  SelfLink:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/PizzaResource"
+  PizzaResult:
     properties:
-      self:
+      data:
+        $ref: "#/definitions/PizzaResource"
+  PizzaResource:
+    properties:
+      type:
         type: string
-        format: url
-        description: Self-link of current resource
-  PaginationLinks:
+        description: the type of the resource (pizza)
+        example: pizza
+      id:
+        type: string
+        description: a unique ID of a pizza
+        example: abc123
+      attributes:
+        $ref: "#/definitions/PizzaAttributes"
+  PizzaAttributes:
+    type: object
     properties:
-      first:
+      name:
         type: string
-        format: url
-        description: The first page of data
-      last:
+        description: the name of the pizza
+        example: My Favorite Margherita
+      ovenTemp:
+        type: integer
+        description: the temperature to bake the pizza at, in Fahrenheit
+        example: 500
+      bakeTime:
+        type: integer
+        description: the time in minutes to bake the pizza for
+        example: 10
+      specialInstructions:
         type: string
-        format: url
-        description: The last page of data
-      prev:
+        description: any special instructions for baking the pizza
+        example: place the basil on after baking
+  IngredientsResult:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/IngredientResource"
+  IngredientResult:
+    properties:
+      data:
+        $ref: "#/definitions/IngredientResource"
+  IngredientAttributes:
+    type: object
+    properties:
+      name:
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        description: the name of the ingredient
+        example: Sausage
+      type:
```

Comment:
Looks like we can't have any attributes/relationships named "type" or "id": https://jsonapi.org/format/#document-resource-object-fields

Reply:
interesting, can change this to ingredientType or something 👍